articles/mongodb-vs-mysql/index.md

```diff
@@ -0,0 +1,79 @@
+
+Examples of platforms run by MySQL include Linux UNIX and Windows. Examples of web-based applications that use MySQL include Facebook, Twitter, YouTube, etcetera.
+
+### MySQL and MongoDB feature comparison
+
+| Feature | MySQL | MongoDB |
+| --- | --- | --- |
+| Programming language | It is written in C++ and C languages. | It is written in C++. |
+| Access language | Uses [structured query language](https://www.dataversity.net/structured-query-language-sql/#) (SQL). | Uses [unstructured query language](https://www.dataversity.net/unql-a-standardized-query-language-for-nosql-databases/) - JSON.
+ |
+| Database type | It is a relational database management system (RDBMS). | It is a document-oriented database designed for storing, retrieving, and managing document-oriented information. |
+| [Schemas](https://www.tutorialspoint.com/dbms/dbms_data_schemas.htm) (Skelton structure that represents the logic of the database). | It has a strict schema. Before storing anything, one needs to define tables and columns clearly. Also, ensure that every row in the table should have the same column. | It has a dynamic schema. Several documents can be dropped with no relation, only that documents need to be supported by data structures. |
+| Attack risks | It is prone to [SQL injection attacks](https://www.imperva.com/learn/application-security/sql-injection-sqli/) (a vector that uses malicious SQL code). | Less risk of attack. Its design makes it hard to attack. |
+| Organized By | Data is organized by a table, row, and columns. | Data is organized by collection, document, and field classification. |
+| Performance and Speed | It is better for smaller databases and more general solutions. | It is better for handling massive, unstructured data sets as it has in-memory speed. |
+| Website | Mysql.com | Mongodb.com |
+| Developer | Oracle corporation does the ongoing development. | MongoDB, Inc. does the ongoing development. |
+| Scaling | Allows using more powerful database servers (vertical scaling). | Allows adding more database servers (horizontal scaling). |
+| Support provider | It offers [Oracle lifetime support](https://www.mysql.com/support/), though at different versions. Each level offers a knowledge base, maintenance releases, bug fixes, patches, and updates. | Offers Enterprise-Grade Support. It allows flexibility to upgrade to the newer versions. It has unlimited access to security fixes, updates, and many more. |
+| Community of users | It has a large community of users, as it started several years ago. | It has a relatively smaller community than that of MySQL, as it is newer. |
+| Command speed | MySQL is slower than MongoDB in executing the same commands like select, update, and insert. | MongoDB is faster in the execution of commands than MySQL. |
```

Comment:
Saying A is greater than B, and B is lesser than A is the same thing. Avoid redundant comparisons. I want to know which aspect of c++ makes it slower than MongoDB? 

In the above table, avoid such redundancies. Wherever applicable, remove it or base the arguments with facts.